src/SFA.DAS.ApplyService.Domain/Roatp/GatewayPageIds.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SFA.DAS.ApplyService.Domain.Roatp
+{
+    public static class GatewayPageIds
+    {
+        public const string LegalName = "LegalName";
+        public const string TradingName = "TradingName";
+        public const string OrganisationStatus = "OrganisationStatus";
+
+        public const string WebsiteAddress = "WebsiteAddress";
+
+
+        public const string OfficeForStudent = "OfficeForStudents";
+        public const string InitialTeacherTraining = "InitialTeacherTraining";
+        public const string Ofsted = "Ofsted";
+        public const string SubcontractorDeclaration = "SubcontractorDeclaration";
+
+        public const string CCOrganisationCompositionCreditors = "CompositionWithCreditors";
```

Comment:
These criminal compliance page ids can be removed at some point, they're referenced in config now